crates/wit-parser/src/ast/resolve.rs

```diff
@@ -839,6 +842,56 @@ impl<'a> Resolver<'a> {
                     }
                 }
                 ast::InterfaceItem::TypeDef(_) => {}
+                ast::InterfaceItem::Nest(n) => {
+                    if let ast::Nest {
+                        from:
+                            UsePath::Package {
+                                id: package_id,
+                                name,
+                            },
+                        attributes,
+                        docs,
+                    } = n
```

Comment:
Could this block run for all `ast::Nest` items intead of just `UsePath::Package`? I think that means that other `nest` items would be silently ignored here otherwise?